File: examples/availability/retrieve_hosted/retrieve_hosted.py
import os
from bloock import bloock
from bloock.client.record import RecordClient
from bloock.entity.availability.hosted_loader import HostedLoader

if __name__ == "__main__":
    bloock.api_key = os.environ["API_KEY"]

    record_client = RecordClient()

    pid = "publish uuid result"
    record = record_client.from_loader(HostedLoader(pid)).build()

File: examples/availability/retrieve_hosted/retrieve_hosted.php
<?php

require "./vendor/autoload.php";

use Bloock\Bloock;
use Bloock\Client\AvailabilityClient;
use Bloock\Client\RecordClient;
use Bloock\Entity\Availability\HostedLoader;

Bloock::$apiKey = getenv("API_KEY") ?: "";

$recordClient = new RecordClient();
$availabilityClient = new AvailabilityClient();

$id = "publish uuid result";
$record = $recordClient->fromLoader(new HostedLoader($id))->build();
